client/src/Shared/Helpful.jsx

```diff
@@ -28,6 +28,20 @@ const Helpful = ({ origin, id, helpCount }) => {
 
   const theme = useContext(Theme);
 
+    if (isQuestion) {
+      styles.width = '125px;';
+      styles['border-right'] = 'none;';
+    } else {
+      styles.width = '200px;';
+      styles['border-right'] = '1px solid black;';
+    }
+    return styles;
+  });
```

Comment:
This block needs to be removed or used, I think removed because it looks like a duplicate of something above.

Reply:
I just fixed it!